apps/git-pulse/src/routes/index.ts

```diff
@@ -0,0 +1,12 @@
+import SearchSuggestionsHandler from '../handlers/SearchSuggestionsHandler';
+import { Router } from 'express';
+const routes = Router();
+
+
+routes.get('/', (req, res) => {
+  res.send({ message: 'Hello API' });
+});
+
+routes.get('/search-suggestions', SearchSuggestionsHandler);
```

Comment:
Thinking about the future, it is possible that our API grows and we have more modules, in which case it would be good to create our routes thinking about these modules.
Suppose today we will work on our repositories module

So a path for this endpoint could be `/repositories/search-suggestions`
Then we can create for example more `/users`, `/sessions`, `other-modules` module endpoints

Reply:
I've been solved many changes. All files were converted to typescript, using an interface and repository in githubService.ts. Intellij and vscode were setted to automatically add an empty line. imports were sorted as You indicated.  But I could't  modularize these route. Maybe we can do a pair programming session. in order to work in topics like middlewares, routes and the githubService class. I have worked gradually on the components, and the suggested changes have been very important because they're a challenge for me. Thanks.